src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * @return true if the command is mutating the data and false otherwise
```

Comment:
Kudos for adding the header comment. As mentioned during the tutorial, it is better to enforce this method is overriden in all the subclasses. Therefore, can you think of a way to enforce this by editing the method signature? Hint: just one keyword will do.

Reply:
I thought about making the method abstract, but then I realized that there is another method in the same class, "execute" that does follows another scheme.

Reply:
@arturs68 Check out the comment in the original repo. It says about using an abstract keyword for the execute command too. https://github.com/nus-cs2103-AY1718S1/addressbook-level3/blob/master/src/seedu/addressbook/commands/Command.java

Reply:
@arturs68 This is the comment given in the original repo "//Note: it is better to make the execute() method abstract, by replacing the above method with the line below:
    //public abstract CommandResult execute();"